cmd/operator/service.go

```diff
@@ -76,13 +76,13 @@ func getServicesByRegionAndLabel(ctx context.Context, project, region, labelSele
 //
 // If the target configuration does not specify any regions, the entire list of
```

Comment:
fix in a separate PR please:
this is not proper godoc. godocs start with the method's name.